msgeditor/page_functionality.js

```diff
@@ -99,15 +99,15 @@ document.addEventListener("DOMContentLoaded", function() {
     add_butt.addEventListener('click', function() {
         var table = document.getElementById('msg_table');
         var new_row = table.rows[0].cloneNode(true);
-        new_row.cells[0].getElementsByTagName('input')[0].setAttribute('id', `id_input_${table.rows.length}`);
+        let table_rows_length = table.rows.length // Optimization. Btw avoid using "var" to declare variables. -- FalB
```

Comment:
```suggestion
        let table_rows_length = table.rows.length // Optimization. Btw avoid using "var" to declare variables.
```
Nit but putting your name is not necessary, it should be clear from the git blame history anyway who wrote this comment/line